### PR TITLE
Fix isort issues

### LINKED ISSUE #185: Isort conflicts with Black
🐞 **Problem**
Isort and black are formatting import differently in ```coordinates/__init__.py```

Referencing https://github.com/ambv/black/issues/127
Referencing reopened https://github.com/timothycrosley/isort/issues/694

Temporary Solution:
Add # isort: skip in front of the import.
Implemented in https://github.com/einsteinpy/einsteinpy/pull/176

🖥
Issue with the following versions:
```isort v4.3.17```
```black v19.3b0```

🎯 **Goal**
Remove the temporary fix of isort: skip in ```coordinates/__init__.py```

💡 **Possible solutions**
Wait till the Issue is solved in Black or Isort

### PR DESCRIPTION
**Pull request by shreyasbapat**
Fixes #185 

Run `isort --multi-line=3 --trailing-comma --force-grid-wrap=0 --use-parentheses --line-width=88 [ file.py ]`
when not in einsteinpy.

### CODE REVIEW

**Comment by shreyasbapat** [comment]
@ritzvik @alpeshjamgade Now it's all fine :)

**Comment by ritzvik** [comment]
You would need to change yml file of circleci as well

**Comment by shreyasbapat** [comment]
> You would need to change yml file of circleci as well

Why in the world?

**Comment by ritzvik** [comment]
> > You would need to change yml file of circleci as well
> 
> Why in the world?

Ohh Sorry. CircleCi is also getting the options from config file 😆

File: .isort.cfg [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[settings]
+multi_line_output=3
+include_trailing_comma=True
+force_grid_wrap=0
+combine_as_imports=True
+line_length=88
```
Comment (ritzvik) [inline .isort.cfg]: Add a newline here. My OCD behind this. Sorry!

Comment (shreyasbapat) [reply .isort.cfg]: No new line there xD

Comment (ritzvik) [reply .isort.cfg]: And squash em

Comment (shreyasbapat) [reply .isort.cfg]: No squashing either. Config files must have their commits. Incase something wrong happens :'(

Comment (ritzvik) [reply .isort.cfg]: Ohh yes! Black-isort fights fixed I guess

Comment (shreyasbapat) [reply .isort.cfg]: No guessing. It did B)

**Pull request merged**